src/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const { join } = require('path');
+const router = require('./routes/router');
+const {PORT} = require('dotenv');
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({extended:false}));
+app.use(express.static(join(__dirname,'..','public')));
+
+app.set('port',PORT || 5000);
```

Comment:
If you tried to run the application always will run on port `5000`  and will escape the port you set in `.env` file

The right way to use env variables is below:
```js

const { config } = require('dotenv');
config();

const { PORT, ANOTHER_VARIABLES } = process.env;

```